packages/strapi-admin/ee/admin/containers/Roles/CreatePage/Permissions/README.md

```diff
@@ -0,0 +1,377 @@
+# Technical documentation
+
+Here we will cover how the `<Permissions />` component works.
+This component is in charge of managing the permissions of the admin panel, it has 4 children one child for each tab.
+The collection types & single types tabs uses the same component: `<ContentTypes>` similarly, the settings & plugins tab uses the `<Plugins>` component.
+
+The UI uses the layout received from the back-end in order to build the UI.
+
+## Layout shape
+
+```js
+const layout = {
+  conditions: [{ id: 'string', displayName: 'string', category: 'string'}], // Array of conditions that can be applied on a permission
+  sections: {
+    plugins: [
+      { displayName: 'string', action: 'string', subCategory: 'string', plugin: 'string'}
```

Comment:
What does action refers to? CRUD?

Reply:
For the content types permissions, actions can be CRUD + publish but for plugins and settings sections, it depends. Settings/plugins can have custom actions.


![Capture d’écran 2021-02-15 à 10 10 03](https://user-images.githubusercontent.com/7756284/107926558-2905c480-6f76-11eb-9751-776eb800d1e4.png)

Reply:
Exactly! But only for the content manager, so for the Collection Types and Single Types tab